frontend/layouts/unified.vue

```diff
@@ -1,5 +1,6 @@
 <template>
     <div :class="isDarkMode && 'dark bg-gray-900'">
+        <StagingHeader v-if="isStagingEnvironment" />
         <Header
```

Comment:
I also suggest you to add it to the Header component instead, so it will be displayed everywhere where there is a header (in the future)